tutorials/adapt/README.md

```diff
@@ -0,0 +1,79 @@
+# Domain adaptation of NMT models
+
+Although the quality of machine translation systems is nowadays remarkably
+good, sometimes it is important to specialize the MT output to the specifics of
+certain domains. These customizations may include preferring some word
+translation over others or adapting the style of the text, among others. In
+this tutorial, we show two methods on how to perform domain adaptation of a
+general translation system using Sockeye.
+
+We assume you already have an already trained Sockeye model, for example the
+one trained in the [second tutorial](../wmt/README.md). We also assume that you
+have two training sets, one composed of general or out-of-domain data, and one
+composed of in-domain data on which you want to adapt your system. Note that
+both datasets need to be pre-processed in the same way.
+
+## Preparing the data
+
+First, you must be careful to prepare the in-domain training data using the same
+vocabulary as the out-of-domain data. Assuming your prepared OOD data resides
+in `ood_data`
+
+    python -m sockeye.prepare_data \
+        -s data/id.train.src.bpe \
+        -t data/id.train.trg.bpe \
+        -o id_data \
+        --source_vocab ood_data/vocab.src.0.json \
+        --target_vocab ood_data/vocab.trg.0.json
+
+Note: If your in-domain data is small, you may skip this step and add the
+corresponding arguments to the `sockeye.train` calls.
+
+## Continuation of training
+
+This method consists in taking the parameters of an already trained system and
```

Comment:
maybe just: 'This method fine-tunes a trained model and starts a second training run on in-domain data, initialized with the parameters obtained from the out-domain data.'